coral/src/app/features/components/filters/SearchConnectorFilter.tsx

```diff
@@ -0,0 +1,13 @@
+import { SearchFilter } from "src/app/features/components/filters/SearchFilter";
+
+function SearchConnectorFilter() {
```

Comment:
I'm not convinced these `Search{entity}Filter` components are necessary? We could directly use `SearchFilter` where needed and inline the placeholder and description?

I understand that they are used in two spots though 🤔 Maybe have an `entity` prop that can be `topic | connector`, and assign the correct placeholder inside `SearchFilter`? So we don't have to duplicate the copy everywhere, but we also don't need such thin abstractions. Like this (very pseudocode): 

```ts

// In SearchFilter

const variantText = {
  topic: {placeholder: "...", description: "..."},
  ...
}

...

placeholder={variantText[props.entity]}

```

Reply:
We can only have one component, the downside would be that we have to pass placeholder as well as the description in all places, which makes it more likely that we diverge here. 

We could pass props to the component for all three cases. I've decided for the wrapper approach because I kind of like it :D  I'm not a super big fan of passing a lot of props 😬  especially because then we need to add more tests -> not that I don't like writing test, but it shows that there are more possible points of failure. 

But I don't have too strong opinions about that, other than I think it's easier to maintain, so open to change it. Maybe @roope-kar  can chime in and tip the scale!

Reply:
> We can only have one component, the downside would be that we have to pass placeholder as well as the description in all places, which makes it more likely that we diverge here.

Yes, that is the concern my second proposition tries to address, with the `entity` variant prop ^^

And I think my position on variant prop VS variant component depends on how different the component rendered is. For buttons, both seem ok to me, with a slight preference for the component variant `PrimaryButton`


```tsx
<Button primary />

<PrimaryButton />
```

But for a component which renders the same UI, but with different text, I feel like it's too big of a box to hold such slight changes :o

Reply:
> I think either way is correct in this case (sorry). 

No need to be sorry 😅  I think we all agree that this is a thing that is based on preference and not actual technical reasons. 

@mathieu-anderson  would you be ok to go with the wrapper solution for now? We can always go back and change when something comes up etc.

Reply:
`SearchFilter.Connector` I would like better though, but let's not prolong this entertaining but unessential conversation hahaha